sdk/python/tests/integration/feature_repos/repo_configuration.py

```diff
@@ -141,6 +142,7 @@
         }
 
 
+# Replace online store contaniers
 if os.getenv("FEAST_LOCAL_ONLINE_CONTAINER", "False").lower() == "true":
```

Comment:
We should document these flags somewhere.. yeesh

Reply:
maybe i guess. I'd prefer people just use the Makefile commands